### PR TITLE
add bootstrap repository definition for SLE15 SP6

### PR DESCRIPTION
**Pull request by mcalmer**
## What does this PR change?

- added SLE15 SP6  to bootstrap data
- added Uyuni Client Tools Channels to spacewalk-common-channels.ini
- add also openSUSE Leap 15.6 and its uyuni client tools

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [x] **DONE**

## Test coverage
- No tests: already covered

- [x] **DONE**

## Links

Issue(s): https://github.com/SUSE/spacewalk/issues/23476
Port(s): https://github.com/SUSE/spacewalk/pull/23814

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

File: susemanager/susemanager.changes.mcalmer.bootstrap-repos-2024 [hunk 1]
```diff
@@ -0,0 +1,2 @@
+- add bootstrap repository definition for openSUSE Leap 15.6
+- add bootstrap repository definition for SLE15 SP6
```
Comment (juliogonzalez) [inline susemanager/susemanager.changes.mcalmer.bootstrap-repos-2024:2]:
```suggestion
- add bootstrap repository definition for SUSE Linux Enterprise 15 SP6
```
At least on the changelog I guess we should use the full name.

File: utils/spacewalk-utils.changes.mcalmer.bootstrap-repos-2024 [hunk 2]
```diff
@@ -0,0 +1,2 @@
+- add openSUSE Leap 15.6 and its client tools
+- add uyuni client tools for SLES 15 SP6
```
Comment (juliogonzalez) [inline utils/spacewalk-utils.changes.mcalmer.bootstrap-repos-2024:2]:
```suggestion
- add Uyuni client tools for SUSE Linux Enterprise 15 SP6
```

**Review by juliogonzalez — APPROVED** [review]
LGTM, assuming PDIDs and list of repos for Leap 15.6 are correct (I didn't tried my self yet).

Just two small suggestions about the changelogs, in case you want to consider them.

**Review by m-czernek — APPROVED** [review]
:+1: LGTM

**Pull request merged**